frontend/src/time_utils.js

```diff
@@ -1,9 +1,24 @@
-/*
+/**
  * Creates Date without a time from Timestamp model.
  * Note: timestampDate.date.month is from 1-12, while JavaScript 'Date' is from 0-11.
  */
 export function convertTimestampToDate(timestampDate) {
   return new Date(
     timestampDate.date.year, timestampDate.date.month - 1, timestampDate.date.day, 0, 0, 0, 0)
-    .toDateString();
+    .toDateString().slice(4);
```

Comment:
does this `slice(4)` mean you don't want the day of the week to show?

Reply:
yup haha